libs/dal/src/repositories/environment/environment.schema.ts

```diff
@@ -27,6 +27,10 @@ const environmentSchema = new Schema(
         },
       },
     ],
+    encrypted: {
```

Comment:
A small suggestion for naming: 
```suggestion
    hmacEncryption: {
```

Since encryption is quite a general term :)

Reply:
I tried to be general, however, I see I was too general. 
I will update to notificationCenterEncryption on the next commit